dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -128,6 +128,11 @@ session.activeSessions.description=The number of concurrently active sessions. (
 session.invalidated.total.description=The number of sessions that have logged out since this metric was enabled.
 session.invalidatedbyTimeout.total.description=The number of sessions that have logged out by timeout since this metric was enabled.
 
+requestTiming.requestCount.description=The number of servlet requests since the server started.
+requestTiming.activeRequestCount.description=The number of servlet requests currently running.
+requestTiming.slowRequestCount.description=The number of servlet requests that are currently running and are deemed slow.
+requestTiming.hungRequestCount.description=The number of servlet requests that are currently running and are deemed hung.
```

Comment:
Suggested change: Replace "and are deemed hung" with "but are hung" or "but are unresponsive"

Reply:
fixed